build.gradle

```diff
@@ -173,6 +176,8 @@ dependencies {
   compile group: 'io.springfox', name: 'springfox-swagger2', version: versions.springfoxSwagger
 
   compile group: 'uk.gov.hmcts.reform', name: 'java-logging', version: versions.reformLogging
+  compile group: 'uk.gov.hmcts.reform', name: 'java-logging-spring', version: versions.reformLogging
```

Comment:
might as well fix warnings in that project:
```java
FilterRegistrationBean<RequestIdsSettingFilter> filterRegistrationBean = new FilterRegistrationBean<>();
```

instead of

```java
FilterRegistrationBean filterRegistrationBean = new FilterRegistrationBean();
```

Reply:
Perhaps this area of spring framework is lacking an upgrade. I'd rather stick in as many filters as I want instead of creating beans here and there

Reply:
we don't actually want java-logging-spring really, it predates app insights